includes/settings/js/src/components/Settings.jsx

```diff
@@ -0,0 +1,96 @@
+/* global atlasContentModeler */
+import React, { useState } from "react";
+import { __ } from "@wordpress/i18n";
+import { dispatch } from "@wordpress/data";
+
+export default function Settings() {
+	const [usageTracking, setUsageTracking] = useState(
+		atlasContentModeler.usageTrackingEnabled
+	);
+
+	function saveUsageTrackingSetting(event) {
+		// @todo catch save errors
+		dispatch("core").saveSite({
+			atlas_content_modeler_usage_tracking: event.target.value,
+		});
+		setUsageTracking(event.target.value);
+	}
+
+	return (
+		<div className="app-card settings-view">
+			<section className="heading flex-wrap d-flex flex-column d-sm-flex flex-sm-row">
+				<h2>{__("Settings", "atlas-content-modeler")}</h2>
+			</section>
+			<section className="card-content">
+				<form>
+					<div className="row">
+						<div className="col-xs-10 col-lg-4 order-1 order-lg-0">
+							<div className="row">
+								<div className="col-xs-12">
+									<h4>
+										{__(
+											"Analytics",
+											"atlas-content-modeler"
+										)}
+									</h4>
+									<p className="help">
+										{__(
+											"Opt into anonymous usage tracking to help us make Atlas Content Modeler better.",
+											"atlas-content-modeler"
+										)}
+									</p>
+									<div className="row">
+										<div className="col-xs-12">
+											<label
+												className="radio"
+												htmlFor="atlas-content-modeler-settings[usageTrackingDisabled]"
+											>
+												<input
+													type="radio"
+													id="atlas-content-modeler-settings[usageTrackingDisabled]"
+													name="atlas-content-modeler-settings[usageTracking]"
+													value="0"
+													checked={
+														usageTracking === "0"
+													}
```

Comment:
This isn't saving settings quite right for me at the moment:

- Disabled isn't active by default (i.e. when atlas_content_modeler_usage_tracking is empty).
- Clicking 'disabled' then refreshing shows no option as active.

This change seems to fix both issues:

```suggestion
													checked={
														usageTracking === "0" ||
														!usageTracking
													}
```